src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,127 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private int listSize;
+    private T[] data = (T[]) new Object[DEFAULT_CAPACITY];
+
+    private void indexExceptionCheck(int index) {
```

Comment:
`private void checkIndex(int index)`
Private ones (methods, classes) should be after public ones.

Reply:
fixed